dvc/utils/__init__.py

```diff
@@ -420,3 +420,8 @@ def resolve_output(inp, out):
     if os.path.isdir(out):
         return os.path.join(out, name)
     return out
+
+
+def is_exec_found(exec_name):
```

Comment:
No need to put it into utils, it is only used in one place.

Reply:
i made it this way because i want to be sure it's tested on windows.
but i can remove it

Reply:
@xliiv What do you mean by "tested on windows"?  That piece of code will be executed on windows too, no?

Reply:
We need to figure out if `less` is installed on every platform.
So yes, it will be running on Windows

Reply:
@xliiv Sorry, I still don't quite understand what `i made it this way because i want to be sure it's tested on windows.` was about.

Reply:
1. we use Travis for CI
2. I see Windows here https://travis-ci.com/iterative/dvc/builds/138856287
So i assumed that tests run on Windows.
If they run on Windows, i thought that it would good to wrap
```
    cmd = "({}) 2>/dev/null".format(exec_name)
    hasattr(os, "system") and os.system(cmd) == 0
```
with a function, write a test for the fn. and make sure it's tested during the CI.
Does it help?

Reply:
if you intend to run this on windows, then use `os.devnull` instead of `"/dev/null"`

Reply:
@casperdcl `os.devnull` is used, thank you!

Reply:
should i resolved this comment?